macros/sql/star.sql

```diff
@@ -1,8 +1,8 @@
-{% macro star(from, relation_alias=False, except=[], prefix='', suffix='') -%}
-    {{ return(adapter.dispatch('star', 'dbt_utils')(from, relation_alias, except, prefix, suffix)) }}
+{% macro star(from, relation_alias=False, except=[], regex='', prefix='', suffix='') -%}
+    {{ return(adapter.dispatch('star', 'dbt_utils')(from, relation_alias, except, regex, prefix, suffix)) }}
 {% endmacro %}
 
-{% macro default__star(from, relation_alias=False, except=[], prefix='', suffix='') -%}
+{% macro default__star(from, relation_alias=False, except=[], regex='', prefix='', suffix='') -%}
```

Comment:
I'd prefer to add the optional `regex` argument to the end of the signature. That way, this new feature will be guaranteed to be non-breaking for all users.

Do you have any thoughts about the order of parameters?

I think the change would be as simple as the following:

```sql
{% macro star(from, relation_alias=False, except=[], prefix='', suffix='', regex='') -%}
    {{ return(adapter.dispatch('star', 'dbt_utils')(from, relation_alias, except, prefix, suffix, regex)) }}
{% endmacro %}

{% macro default__star(from, relation_alias=False, except=[], prefix='', suffix='', regex='') -%}
```